package.json

```diff
@@ -5,6 +5,7 @@
   "scripts": {
     "eslint": "eslint --ignore-path .gitignore .",
     "eslint-fix": "eslint --fix --ignore-path .gitignore .",
+    "stylelint": "stylelint --ignore-path .gitignore .",
```

Comment:
We need to hook this into our `test` script, or people won't run it, and it won't happen in CI automatically.

There are a few ways to do it.  Here's one way, but adding a `lint` and `postlint` script (NOTE: I also changed the `pretest` to use lint vs. eslint):

```
"stylelint": "stylelint --ignore-path .gitignore .",
"lint": "npm run eslint",
"postlint": "npm run stylelint",
"pretest": "npm run lint",
```

Another way would be to run parallel scripts using `npm-run-all` like they do in https://github.com/stylelint/stylelint/blob/master/package.json.

Either way is probably fine.  Our goal is for `npm test` to do this:

- eslint
- stylelint
- jest

Once you do this, we should start failing Travis due to our stylesheet having errors.